src/App.tsx

```diff
@@ -52,13 +52,35 @@ import ErrorPage from "./components/Errors/ErrorPage";
 import { useQuery } from "@tanstack/react-query";
 import { AxiosError } from "axios";
 import { Provider } from "jotai";
+import { permissionService } from "./services/permissions/permissionsService";
+import { RESOURCES } from "./services/permissions/resources";
```

Comment:
Use CamelCase for variable names

Reply:
It is a constant so I went with all caps